blocklylib-core/src/main/java/com/google/blockly/android/AbstractBlocklyActivity.java

```diff
@@ -684,6 +690,24 @@ public void onClick(View v) {
     }
 
     /**
+     * This method finds and set the visibility of {@link R.id#blockly_zoom_in_button} and
+     * {@link R.id#blockly_zoom_out_button}
+     * from the view hierarchy according
+     * <p/>
+     */
+    protected void onConfigureZoomBehavior(){
```

Comment:
One other nit, please have onConfigureZoomBehavior create and return the ZoomBehavior object, similar to onCreateBlockViewFactory